viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move
+
+
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie_dict= {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
 
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"]== title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+        
+    return user_data

```

Comment:
This works in this particular case because we only expect the user to have one copy of each movie in their watchlist. But in general, try to avoid modifying a data structure as we're doing a for loop over it; very strange and hard to debug behavior can result. Consider instead using the loop to identify what needs to be removed, then remove it after the loop.